charts/tempo-distributed/templates/query-frontend/servicemonitor-jaeger-query.yaml

```diff
@@ -1,51 +1 @@
-{{- with .Values.serviceMonitor }}
-{{- if .enabled }}
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
-metadata:
-  name: {{ template "tempo.queryFrontendFullname" $ }}-tempo-query
-  {{- with .namespace }}
-  namespace: {{ . }}
-  {{- end }}
-  {{- with .annotations }}
-  annotations:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
-  labels:
-    {{- include "tempo.queryFrontendLabels" $ | nindent 4 }}
-    {{- with .labels }}
-    {{- toYaml . | nindent 4 }}
-    {{- end }}
-spec:
-  {{- with .namespaceSelector }}
-  namespaceSelector:
-  {{- toYaml . | nindent 4 }}
-  {{- end }}
-  selector:
-    matchLabels:
-      {{- include "tempo.queryFrontendSelectorLabels" $ | nindent 6 }}
-  endpoints:
-    - port: tempo-query-metrics
-      {{- with .interval }}
-      interval: {{ . }}
-      {{- end }}
-      {{- with .scrapeTimeout }}
-      scrapeTimeout: {{ . }}
-      {{- end }}
-      {{- with .relabelings }}
-      relabelings:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
-      {{- with .metricRelabelings }}
-      metricRelabelings:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
-      {{- with .scheme }}
-      scheme: {{ . }}
-      {{- end }}
-      {{- with .tlsConfig }}
-      tlsConfig:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
-{{- end }}
-{{- end }}
+{{- include "tempo.lib.serviceMonitor" (dict "ctx" $ "component" "tempo-query") }}
```

Comment:
Is this really needed? Isn't the frontend covered by query-frontend service? I'd test this and check for double metrics!